txndata/constraints.lisp

```diff
@@ -386,18 +386,22 @@
 ;;                          ;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-(defconstraint   cumulative-gas ()
-                 (begin (if-zero ABS
-                                 (vanishes! GAS_CUMULATIVE))
+(defconstraint   cumulative-gas---vanishing-in-padding ()
```

Comment:
it doesn't look the standard formatting for constraints, it's the formatting for shorthand

Reply:
We've written many constraints this way, not just shorthands.